File: app/src/main/java/com/gb/students/crm_task_manager/model/entity/contact/Relation.java
package com.gb.students.crm_task_manager.model.entity.contact;

import java.util.Date;

// информация об отношениях, романтические, родственные ...
public class Relation {
    private String type;
    private String name;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Date getBirth() {
        return birth;
    }

    public void setBirth(Date birth) {
        this.birth = birth;
    }

    private Date birth;

}

File: app/src/main/java/com/gb/students/crm_task_manager/model/entity/contact/Relation.kt
package com.gb.students.crm_task_manager.model.entity.contact

import com.gb.students.crm_task_manager.custom.StringHelper
import java.util.*

class Relation {
    var type: String? = null
    var name: String? = null
    var birth: Date? = null
    var note: String? = null

    override fun toString(): String {
        return "$name, $type, ${StringHelper.getDateFormat(StringHelper.Pattern.DOT_NUMERIC).format(birth)}\n$note"
    }
}